### PR TITLE
MAINTAINERS: various enhancements

### PR DESCRIPTION
**Pull request by nashif**
Some platform areas did not have the bindings associated with that
platform.

Signed-off-by: Anas Nashif <anas.nashif@intel.com>

### CODE REVIEW

**Review by galak — COMMENTED** [review]
Are we ok w/false positives in some of these cases.  For example `intel,` could apply to x86 as well.  `microchip,` has non-EC devices supported in tree.

**Comment by nashif** [comment]
> Are we ok w/false positives in some of these cases. For example `intel,` could apply to x86 as well. `microchip,` has non-EC devices supported in tree.

better than not geeting any reviewers at all, false positive are not fatal in this case IMO, should be fine.

**Review by galak — DISMISSED** [review]

File: MAINTAINERS.yml [hunk 1]
```diff
@@ -1929,6 +1930,7 @@ NXP Platforms:
     - dts/arm/nxp/
     - dts/bindings/*/nxp*
     - soc/xtensa/nxp_adsp/
+    - dts/bindings/*/nxp,*
```
Comment (MaureenHelm) [inline MAINTAINERS.yml:1933]: this looks redundant to two lines up

**Review by jhedberg — DISMISSED** [review]

**Review by MaureenHelm — DISMISSED** [review]

**Review by stephanosio — DISMISSED** [review]

**Review by fabiobaltieri — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Review by mbolivar-nordic — APPROVED** [review]

**Review by galak — APPROVED** [review]

**Review by stephanosio — APPROVED** [review]

File: MAINTAINERS.yml [hunk 2]
```diff
@@ -1902,6 +1902,7 @@ Intel Platforms (Xtensa):
     - soc/xtensa/intel_*/
     - dts/xtensa/intel/
     - tests/boards/intel_adsp/
+    - dts/bindings/*/intel,*
   labels:
     - "platform: Intel ADSP"
```
Comment (finikorg) [inline MAINTAINERS.yml:1983]: @nashif now all intel drivers get added to "platform: Intel ADSP".

**Pull request merged**